BioSQL/BioSeqDatabase.py

```diff
@@ -595,16 +595,11 @@ def get_subseq_as_string(self, seqid, start, end):
         #    """select SUBSTRING(seq FROM %s FOR %s)
         #             from biosequence where bioentry_id = %s""",
         #    (start+1, length, seqid))[0]
-        #
-        # Convert to a string on returning for databases that give back
-        # unicode. Shouldn't need unicode for sequences so this seems safe.
-        return str(
-            self.execute_one(
-                """select SUBSTR(seq, %s, %s)
+        return self.execute_one(
+            """select SUBSTR(seq, %s, %s)
                      from biosequence where bioentry_id = %s""",
-                (start + 1, length, seqid),
-            )[0]
-        )
+            (start + 1, length, seqid),
+        )[0]
```

Comment:
The indentation of the multiline string looks odd, it would fit as a single line string with the black line length of 88:

```python
            "select SUBSTR(seq, %s, %s) from biosequence where bioentry_id = %s",
```

Also most of the SQL uses upper case as in:

```python
            "SELECT SUBSTR(seq, %s, %s) FROM biosequence WHERE bioentry_id = %s",
```